### DISCUSSION TITLE
blur taskbar when maximize window

### DISCUSSION THREAD

**Discussion by BMend-H** [ramensoftware/windhawk-mods#1050] [Q&A]
Using Windows 11 Taskbar Styler and the taskbar becomes transparent/blurry on the desktop. 
When I open a window it stays transparent/blurry with the content of the window, without showing the desktop. However, when I maximize the window it doesn’t stay transparent/blurry with the content of the window instead, it becomes transparent/blurry showing the desktop. How can I maximize the window and have the taskbar remain transparent/blurry displaying the content of the window?

I’ve already tried using Taskbar Background Helper, but it didn’t help.

**Comment by m417z** [comment]
There's no mod to do that. And it's not so easy to implement, unless you just resize the window to be covered by the taskbar, in which case the bottom part of the window won't be accessible.
